docs/4.0. Behaviour.md

```diff
@@ -226,40 +230,64 @@ Each activation is allocated a unique identifier by the API. This identifier is
+An HTTP 200 response MUST be returned when an immediate activation has been successfully requested. For a request with an immediate activation the API SHOULD only return a response once the new map has been applied to the audio.
 
-An HTTP 202 response SHOULD be returned when a request for a scheduled activation is accepted, to indicate that while the request itself was acceptable it has not yet been acted upon by the sender. For scheduled activations `activation_time` should be the absolute TAI time the parameters will actually transition. For absolute scheduled activations SHOULD be the same as the requested time, but may differ if the Device is unable to schedule at the requested time.
+An HTTP 202 response MUST be returned when a request for a scheduled activation is accepted, to indicate that while the request itself was acceptable it has not yet been acted upon by the sender. For scheduled activations `activation_time` SHOULD be the absolute TAI time the parameters will actually transition. For absolute scheduled activations SHOULD be the same as the requested time, but may differ if the Device is unable to schedule at the requested time.
 
-If an Output is modified by an already scheduled activation the API SHOULD respond with HTTP 423 `resource locked`. The API SHALL reject the entire request, even if some of the routes in the failed request are not currently locked by another activation.
+In there is an error in the request, the API MUST reject the entire request with an appropriate 4xx response. Particular responses MUST be used in these cases:
 
-Any attempt to set one of the two fields within a Output channel object to `null` without setting the other SHOULD result in an HTTP 400 response. In this error scenario the API MUST reject the activation request in full, even if some of the routes in the failed request would have been valid.
+| Error | HTTP Response |
+| --- | --- |
+| Output is modified by an already scheduled activation. | 423 (Locked) |
+| Routing request not permitted by Input or Output constraints | 400 (Bad Request). |
+| One field of Output channel object is `null` but other is not. | 400 (Bad Request) |
 
-In the event that the stage request attempts to perform a routing operation that is not permitted by the Input or Output routing constraints the API SHOULD respond with an HTTP 400 error response. The API SHALL reject the entire request, even if some of the routes in the failed request would have been valid.
 
-Note that for activations the `mode` parameter MUST return to null once the activation transaction with the client is completed (i.e the response to the PATCH has been sent by the server).
+For activations the `mode` parameter MUST return to `null` once the activation transaction with the client is completed (i.e the response to the PATCH has been sent by the server).
```

Comment:
So far as I can see the API doesn't use PATCH requests on any endpoint, I assume this sentence should be referring to POST requests. But POST requests are also mentioned in the Overview and in the schemas, so I'll raise a separate issue.

Reply:
Moved to issue #43